src/test/scala/chiselTests/util/CatSpec.scala

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests.util
+
+import chisel3._
+import chisel3.stage.ChiselStage
+import chisel3.util.Cat
+
+import chiselTests.ChiselFlatSpec
+
+object CatSpec {
+
+  class JackIsATypeSystemGod extends MultiIOModule {
```

Comment:
😑

Reply:
I knew the type system god.  The type system god was a friend of mine.  Senator, you're no type system god.